packages/logger/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+    "include": ["src/**/*.ts"],
+    "compilerOptions": {
+        "declaration": true,
+        "outDir": "dist",
+        "sourceMap": true,
+        "baseUrl": "./src",
+        "resolveJsonModule": true,
+        "lib": ["ESNext"]
```

```suggestion
        "lib": ["2019"]
```
safer for node 12 support.

---

Fixed in https://github.com/SAP/open-ux-tools/pull/257/commits/c2348bd8d23c4b47495add3e57d3881c8eb5241f